.github/workflows/go.yml

```diff
@@ -0,0 +1,71 @@
+name: Go Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    strategy:
```

Comment:
try ```max-parallel: 1``` after strategy instead of ledegrname+random number

Reply:
Done.